common/src/test/java/org/keycloak/common/ProfileTest.java

```diff
@@ -148,30 +145,6 @@ public void enablePreviewWithCommaSeparatedList() {
         Assert.assertTrue(Profile.isFeatureEnabled(PREVIEW_FEATURE));
     }
 
-    @Test
-    public void enablePreviewWithPropertiesFile() throws IOException {
-        Properties properties = new Properties();
-        properties.setProperty("profile", "preview");
```

Comment:
We will not support this?

Reply:
I'm advocating that we not. We don't need 3 different ways of specifying these options.

Reply:
Just to be sure; I haven't noticed any removal of this logic in the codebase, so I assume it is still working and users are not required to migrate their props files, right?

Reply:
It was used only for testing, and was not documented for general use.

Reply:
Ok, thanks for the info. Works for me then.

Reply:
Yeah, if we can get rid of not using that property file it will make things simpler.